modules/installation-configuration-parameters.adoc

```diff
@@ -1437,6 +1437,14 @@ link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
 |The link:https://cloud.google.com/compute/docs/disks#disk-types[GCP disk type].
 |Either the default `pd-ssd` or the `pd-standard` disk type. The control plane nodes must be the `pd-ssd` disk type. Compute nodes can be either type.
 
+|`platform.gcp.defaultMachinePlatform.osImage.project`
+|Optional. By default, the installation program downloads and installs the {op-system} image that is used to boot control plane and compute machines. You can override the default behavior by specifying the location of a custom {op-system} image that the installation program is to use for both types of machines.
+|String. The name of GCP project where the image is located.
+
+|`platform.gcp.defaultMachinePlatform.osImage.name`
```

Comment:
Are these supposed to be alphabetical order? If so, then this should come before the above one

Reply:
No. These are not supposed to be in alphabetical order.